roles/deploy_shinken_server/README.md

```diff
@@ -0,0 +1,49 @@
+Deploy Shinken Server
+=========
+
+This is an Ansible Role to Deploy Shinken Monitoring Server with Basic installation.
```

Comment:
Perhaps describe a bit what the basic installation constitutes (i.e. installs packs for vmware, rhv, linux, etc.), and maybe provide a link to the Shinken documentation so the user knows where to go after the installation completes (and how to make use of the packs).